app/assets/stylesheets/application.scss

```diff
@@ -61,30 +70,24 @@ $indianred: #e45e56;
   text-align: center;
 }
 
-// Footer
-
-.page-wrap {
-	margin-bottom: -142px;
-  min-height: 100%;
-}
-
-.page-wrap:after {
-  content: "";
-  display: block;
-}
+$footer-height: 100px;
 
-.site-footer,
-.page-wrap:after {
-  height: 142px;
+.wrapper {
+  min-height: calc(100% - #{$footer-height});
```

Comment:
Properties should be ordered background-image, background-position, background-position, background-repeat, background-size, min-height